src/daemon/CMakeLists.txt

```diff
@@ -454,8 +454,11 @@ elseif ("${CKB_NEXT_INIT_SYSTEM}" STREQUAL "unknown")
 endif ()
 
 if (LINUX)
+  if (SAFE_INSTALL)
+    install(CODE "file(REMOVE \"/etc/udev/rules.d/99-ckb-daemon.rules\")")
```

Comment:
At first I thought it's a clever way to migrate a user after breaking backwards compatibility.

However, depending on the resolution of #311 `/etc` is supposed to contain user-made udev rules, so this line cannot persist, since it would erase them upon each potential reinstallation.

Reply:
We could hash the file in its current state, and if it matches the hash, then cmake removes it and migrates it.

Reply:
Good idea, `file(<MD5|SHA1|SHA224|SHA256|SHA384|SHA512> <filename> <variable>)` can do it.

Reply:
I can't edit this request right now as I'm on holiday presently with limited internet access. Do you need me to do anything as I'm currently not back till mid January.

Reply:
@Kedstar99 don't worry, we can take care of it. Apologies for holding this for so long